<<<<<<< HEAD
From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
=======
From 559fbff13af9ce2fbc0b9bc5727a7323e1db6217 Mon Sep 17 00:00:00 2001
>>>>>>> 712839d4c6 (Removed unwanted submodules from index)
From: Eneas U de Queiroz <cote2004-github@yahoo.com>
Date: Thu, 27 Sep 2018 08:29:21 -0300
Subject: Do not use host kernel version to disable AFALG

This patch prevents the Configure script from using the host kernel
version to disable building the AFALG engine on openwrt targets.

Signed-off-by: Eneas U de Queiroz <cote2004-github@yahoo.com>

<<<<<<< HEAD
--- a/Configure
+++ b/Configure
@@ -1677,7 +1677,9 @@ $config{CFLAGS} = [ map { $_ eq '--ossl-
=======
diff --git a/Configure b/Configure
index 5a699836f3..74d057c219 100755
--- a/Configure
+++ b/Configure
@@ -1548,7 +1548,9 @@ unless ($disabled{"crypto-mdebug-backtrace"})
>>>>>>> 712839d4c6 (Removed unwanted submodules from index)
 
 unless ($disabled{afalgeng}) {
     $config{afalgeng}="";
-    if (grep { $_ eq 'afalgeng' } @{$target{enable}}) {
+    if ($target =~ m/openwrt$/) {
+        push @{$config{engdirs}}, "afalg";
+    } elsif (grep { $_ eq 'afalgeng' } @{$target{enable}}) {
         my $minver = 4*10000 + 1*100 + 0;
         if ($config{CROSS_COMPILE} eq "") {
             my $verstr = `uname -r`;
